### ISSUE TITLE
Loader: vkDestroyDebugReportCallbackEXT is ignored, callbacks still fired and may crash the app

### ISSUE THREAD

**Issue by scyganINTEL** [KhronosGroup/Vulkan-LoaderAndValidationLayers#1035]
Repro: 
1) Run the VulkanCTS with `--deqp-validation=enable` in command line, with `VK_LAYER_LUNARG_standard_validation` set of layers.

2) Observe a callback being fired *after* `vkDestroyDebugReportCallbackEXT` from `vkDestroyDevice` on the same thread, with messages like
```
pMessage = 0x00000060b15eed00 "Unloading layer library ...layers\\.\\VkLayer_threading.dll"
```

As app is no longer expecting any callbacks, this may crash in application code

**Comment by lentinem** [comment]
Sounds like a loader issue.

**Comment by MarkY-LunarG** [comment]
It does sound like a loader issue.  I'll look into it.

**Comment by MarkY-LunarG** [comment]
It is definitely a loader bug.  Working on a fix.  Should be in the tree by end of day.

**Comment by MarkY-LunarG** [comment]
Fixed with 80aa1b733b81aed156972155d262b0da46d18ab2
